forge-gui/res/cardsfolder/upcoming/deathcap_marionette.txt

```diff
@@ -0,0 +1,9 @@
+Name:Deathcap Marionette
+ManaCost:1 B
+Types:Creature Fungus
+PT:1/1
+K:Deathtouch 
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | OptionalDecider$ True | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When CARDNAME enters the battlefield, you may mill two cards. (You may put the top two cards of your library into your graveyard.)
+SVar:TrigMill:DB$ Mill | Defined$ You | NumCards$ 2 
```

Comment:
@tool4ever @Northmoc 
the "you may mill two" , i don't know if that is a rules change,
but the release notes say this:

> If you don't have at least two cards in your library, you can't choose to mill two cards.

meaning we can't use Optional Trigger there?
or does Mill Effect can handle this?

for example, `Crawling Sensation` doesn't have the ruling yet?

Reply:
it's been in the CR for a while and we support it since 62bc1b4

Reply:
ah, so they just didn't update the ruling on the cards